venia-integration-tests/src/actions/wishlist/index.js

```diff
@@ -0,0 +1,74 @@
+import {
+    wishlistPageHeading,
+    wishlistRoot,
+    wishlistItemPrice,
+    wishlistItemAddToCartSection,
+    wishlistItemAddToCartButton,
+    wishlistItemMoreActionsButton,
+    removeProduct,
+    deleteProduct
+} from '../../fields/wishlist';
+import { wishlistRemove, removeProductMessage } from '../../fixtures/wishlist';
+
+/**
+ * Utility function to assert empty wishlist
+ * @param {String} wishlistHeaderText wishlist page header text 
+ */
+export const assertWishlistHeading = (wishlistHeaderText) => {
```

Comment:
I am thinking about moving these assertions to a folder of their own. Ill make the changes and push the commit.